packages/web/src/app/keywords.css

```
@@ -0,0 +1,181 @@
+/*
+ * Keyword rhombus badges — `[Accelerate]`, `[Reaction]`, …
+ *
+ * Sides sit at 80° to the base (10° off vertical). Beaufort for LoL is loaded
+ * in app/layout.tsx as --font-beaufort.
+ */
+
+.card-keyword {
+  /* tan(10°) ≈ 0.176 — horizontal inset for an 80° parallelogram at 1em height */
+  --keyword-inset: 0.2em;
+  display: inline-flex;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)